src/main/webapp/WEB-INF/views/hello.jsp

```diff
@@ -0,0 +1,9 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
```

Comment:
For future - whenever you make PR, it's better to push only the code that is related to functional, that has been implemented